chain_bridge.go

```diff
@@ -87,13 +87,13 @@ func (l *LndRpcChainBridge) GetBlock(ctx context.Context,
 func (l *LndRpcChainBridge) GetBlockHeader(ctx context.Context,
 	hash chainhash.Hash) (*wire.BlockHeader, error) {
 
-	header, err := l.lnd.ChainKit.GetBlockHeader(ctx, hash)
+	block, err := l.lnd.ChainKit.GetBlock(ctx, hash)
```

Comment:
Why is this change needed? This is actually a performance regression, because fetching the full block is always more expensive than fetching the header (especially for Neutrino).

Reply:
I've removed this change, thanks for pointing it out. I think I made this change because I was originally using a local lndclient repo.